spdmlib/src/common/mod.rs

```diff
@@ -1169,11 +1478,14 @@ pub struct SpdmRuntimeInfo {
 pub struct SpdmRuntimeInfo {
     connection_state: SpdmConnectionState,
     last_session_id: Option<u32>,
+    local_used_cert_chain_slot_id: u8,
+    peer_used_cert_chain_slot_id: u8,
     pub need_measurement_summary_hash: bool,
     pub need_measurement_signature: bool,
     pub message_a: ManagedBufferA,
     pub digest_context_m1m2: Option<SpdmHashCtx>, // for M1/M2
     pub digest_context_l1l2: Option<SpdmHashCtx>, // for out of session get measurement/measurement
+    pub digest_context_mut_m1m2: Option<SpdmHashCtx>, // for mutual authentication M1/M2
```

Comment:
`digest_context_mut_m1m2` is for basic mutual auth, which is deprecated.
Why we need it?